### PR TITLE
Doc/update readme

### PR DESCRIPTION
**Pull request by chahank**
Changes proposed in this PR:
- Small update of the readme including a clearer reference list.

This PR fixes #

### PR Author Checklist

- [ ] Read the [Contribution Guide][contrib]
- [ ] Correct target branch selected (if unsure, select `develop`)
- [ ] Descriptive pull request title added
- [ ] Source branch up-to-date with target branch
- [ ] [Documentation](https://climada-python.readthedocs.io/en/latest/guide/Guide_PythonDos-n-Donts.html#2.--Commenting-&-Documenting) updated
- [ ] [Tests][testing] updated
- [ ] [Tests][testing] passing
- [ ] No new [linter issues][linter]
- [ ] [Changelog](https://github.com/CLIMADA-project/climada_python/blob/main/CHANGELOG.md) updated

### PR Reviewer Checklist

- [ ] Read the [Contribution Guide][contrib]
- [ ] [CLIMADA Reviewer Checklist](https://climada-python.readthedocs.io/en/latest/guide/Guide_Reviewer_Checklist.html) passed
- [ ] [Tests][testing] passing
- [ ] No new [linter issues][linter]

[contrib]: https://github.com/CLIMADA-project/climada_python/blob/main/CONTRIBUTING.md
[testing]: https://climada-python.readthedocs.io/en/latest/guide/Guide_Continuous_Integration_and_Testing.html
[linter]: https://climada-python.readthedocs.io/en/stable/guide/Guide_Continuous_Integration_and_Testing.html#3.C.--Static-Code-Analysis

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,13 +55,19 @@ The documentation can also be [built locally](https://climada-python.readthedocs
...
+)
 
-and/or the following published articles:
+- *Uncertainty and sensitivity analysis (unsequa)*:  Kropf, C. M. et al., 2022: Uncertainty and sensitivity analysis for probabilistic weather and climate-risk modelling: an implementation in CLIMADA v.3.1.0. Geoscientific Model Development 15, 7177–7201, [https://doi.org/10.5194/gmd-15-7177-2022](https://doi.org/10.5194/gmd-15-7177-2022
+)
 
-Aznar-Siguan, G. and Bresch, D. N., 2019: CLIMADA v1: a global weather and climate risk assessment platform, Geosci. Model Dev., 12, 3085–3097, https://doi.org/10.5194/gmd-12-3085-2019
+- *LitPop exposures* : Eberenz, S., et al., D. N. (2020): Asset exposure data for global physical risk assessment. Earth System Science Data 12, 817–833, [https://doi.org/10.3929/ethz-b-000409595](https://doi.org/10.3929/ethz-b-000409595)
 
-Bresch, D. N. and Aznar-Siguan, G., 2021: CLIMADA v1.4.1: towards a globally consistent adaptation options appraisal tool, Geosci. Model Dev., 14, 351-363, https://doi.org/10.5194/gmd-14-351-2021
```
Comment (peanutfun) [inline README.md:61]: I suggest moving this into the documentation, maybe on its own page. I think this is too detailed information for a README. There, we can also provide, e.g., appropriate Bibtex entries

File: README.md [hunk 2]
```diff
@@ -15,30 +15,33 @@ CLIMADA is divided into two parts (two repositories):
...
+
 It is **highly recommended** to install CLIMADA into a **separate** Anaconda environment.
 See the [installation guide](https://climada-python.readthedocs.io/en/latest/guide/install.html) for further information.
 
-Follow the [tutorial](https://climada-python.readthedocs.io/en/latest/tutorial/1_main_climada.html) `climada_python-x.y.z/doc/tutorial/1_main_climada.ipynb` in a Jupyter Notebook to see what can be done with CLIMADA and how.
+Follow the [tutorial](https://climada-python.readthedocs.io/en/stable/tutorial/1_main_climada.html) `climada_python-x.y.z/doc/tutorial/1_main_climada.ipynb` in a Jupyter Notebook to see what can be done with CLIMADA and how.
 
 ## Documentation
 
 Documentation is available on Read the Docs:
```
Comment (peanutfun) [inline README.md:36]:
```suggestion
The online documentation is available on Read the Docs: https://climada-python.readthedocs.io/en/stable/

The documentation of each release version of CLIMADA can be accessed separately through the drop-down menu at the bottom of the left sidebar. Additionally, the version 'stable' refers to the most recent release (installed via `conda`), and 'latest' refers to the latest unstable development version (the `develop` branch).
```

File: README.md [hunk 3]
```diff
@@ -15,30 +15,33 @@ CLIMADA is divided into two parts (two repositories):
...
 
-Note that all the documentations has two versions,'latest' and 'stable', and explicit version numbers, such as 'v3.1.1', in the url path. 'latest' is created from the 'develop' branch and has the latest changes by developers, 'stable' from the latest release. For more details about documentation versions, please have a look at [here](https://readthedocs.org/projects/climada-python/versions/).
+Note that all the documentations has two versions,'latest' and 'stable', and explicit version numbers, 'vx.x.x', in the url path. 'stable' is created from the latest release (installed via conda) and 'latest' is created from the 'develop' branch and has the latest changes by developers. For more details about documentation versions, please have a look at [here](https://readthedocs.org/projects/climada-python/versions/).
```
Comment (peanutfun) [inline README.md:38]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -15,30 +15,33 @@ CLIMADA is divided into two parts (two repositories):
 
 It is recommend for new users to begin with the core (1) and the [tutorials](https://github.com/CLIMADA-project/climada_python/tree/main/doc/tutorial) therein.
 
-This is the Python (3.8+) version of CLIMADA - please see https://github.com/davidnbresch/climada for backward compatibility (MATLAB).
+This is the Python (3.8+) version of CLIMADA - please see [https://github.com/davidnbresch/climada](https://github.com/davidnbresch/climada) for backward compatibility (MATLAB).
```
Comment (peanutfun) [inline README.md:18]: The link is already rendered as link? 🤔  Also, we're on Python 3.9 now
```suggestion
This is the Python (3.9+) version of CLIMADA - please see https://github.com/davidnbresch/climada for backward compatibility (MATLAB).
```

File: README.md [hunk 5]
```diff
@@ -6,7 +6,7 @@
...
 
-As of today, CLIMADA provides global coverage of major climate-related extreme-weather hazards at high resolution via a [data API](https://climada.ethz.ch/data-api/v1/docs), namely (i) tropical cyclones, (ii) river flood, (iii) agro drought and (iv) European winter storms, all at 4km spatial resolution - wildfire to be added soon. For all hazards, historic and probabilistic event sets exist, for some also under select climate forcing scenarios (RCPs) at distinct time horizons (e.g. 2040). See also [papers](https://github.com/CLIMADA-project/climada_papers) for details.
+As of today, CLIMADA provides global coverage of major climate-related extreme-weather hazards at high resolution via a [data API](https://climada.ethz.ch/data-api/v1/docs) such as tropical cyclones, river flood, European winter storms, earthquakes, ..., all at 4km spatial resolution - wildfire to be added soon. For all hazards, historic and probabilistic event sets exist, for some also under select climate forcing scenarios (RCPs) at distinct time horizons (e.g. 2040). See also [papers](https://github.com/CLIMADA-project/climada_papers) for details.
```
Comment (peanutfun) [inline README.md:9]:
```suggestion
As of today, CLIMADA provides global coverage of major climate-related extreme-weather hazards at high resolution via a [data API](https://climada.ethz.ch/data-api/v1/docs) such as tropical cyclones, river flood, European winter storms, earthquakes, ..., all at 4km spatial resolution - wildfire to be added soon. For all hazards, historic and probabilistic event sets exist, for some also under select climate forcing scenarios (RCPs) at distinct time horizons (e.g. 2040).
You will find a repository containing scientific peer-reviewed articles that explain software components implemented in CLIMADA here: https://github.com/CLIMADA-project/climada_papers
```

**Review by peanutfun — CHANGES_REQUESTED** [review]
I would like to keep the README as concise as possible. I suggest moving the most important articles to their own documentation page "How to Cite", where we can additionally provide a Bibtex file. I also find a lot of the formulations convoluted and not helpful. This is not due to this PR, but I suggest cleaning things up a bit as we are touching the README now 🙃

**Comment by emanuel-schmid** [comment]
In order to make the links work in the docs _and_ on the GitHub front page, we'll probably have to move both, CITATION_GUIDE.md and REFERENCES.bib to the root directory and symbolically link them within doc/misc.
Analogous to AUTHORS.md, CHANGELOG.md and CONTRIBUTING.md.

**Comment by chahank** [comment]
@peanutfun : do you like the proposal so far?

**Review by peanutfun — CHANGES_REQUESTED** [review]
Content-wise, I agree. But I have a few technical points:
* Why are you using the `[link](link)` pattern? If you don't want to replace the link with text, you can just enter the link. It will be rendered as link automatically.
* There are a few instances where you use a "here" link, which is [highly discouraged](https://webmasters.stackexchange.com/questions/7114/how-to-avoid-click-here-links)
* To avoid the symlink-issue, I suggest making the citation guide an RST file which is only included in the documentation. The link in the README can then point to the appropriate readthedocs documentation page. I can do that.
* Currently, the citation guide document is not included in the Sphinx TOC tree and hence does not appear in the docs. I can fix that.

**Comment by chahank** [comment]
> Content-wise, I agree. But I have a few technical points:
> 
>     * Why are you using the `[link](link)` pattern? If you don't want to replace the link with text, you can just enter the link. It will be rendered as link automatically.
No good reason then. I didn't know this always works. Thanks!
> 
>     * There are a few instances where you use a "here" link, which is [highly discouraged](https://webmasters.stackexchange.com/questions/7114/how-to-avoid-click-here-links)
While I do not fully agree with the strong opinion of this stack exchange, one can change it no problem.
> 
>     * To avoid the symlink-issue, I suggest making the citation guide an RST file which is only included in the documentation. The link in the README can then point to the appropriate readthedocs documentation page. I can do that.
May I ask what you mean by " is only included in the documentation."?
> 
>     * Currently, the citation guide document is not included in the Sphinx TOC tree and hence does not appear in the docs. I can fix that.
Cool thanks!

**Comment by chahank** [comment]
I understand. Thanks for the explanation. Makes sense to me!

**Comment by peanutfun** [comment]
> * To avoid the symlink-issue, I suggest making the citation guide an RST file which is only included in the documentation. The link in the README can then point to the appropriate readthedocs documentation page. I can do that.
> * Currently, the citation guide document is not included in the Sphinx TOC tree and hence does not appear in the docs. I can fix that.

I did both things and brushed up the citation guide a bit.

**Comment by emanuel-schmid** [comment]
🙌 very nice.

**Pull request merged**